test/Test/Basic.hs

```diff
@@ -2,33 +2,16 @@
 -- SPDX-License-Identifier: LicenseRef-MIT-TQ
 
 module Test.Basic
```

Comment:
@rinn7e now that #13 is merged, you can rebase again and cleanup some stuff. For example, this module is not needed anymore.

Reply:
I've removed this as well as using `BaseDAO` contract with the tests, but most of the tests fail pretty much.

I still keep `SampleFA2.hs` just in case we want to make the tests pass. Let me know if you want to change something else, Ivan.

Reply:
> but most of the tests fail pretty much.

They are expected to fail because `transfer` is not implemented yet. But it should be implemented soon, and then hopefully tests will pass for the actual contract.

Reply:
FYI, currently I'm working on `debitFrom` and `creditTo` which is needed for the `propose` entrypoint (freezing the token) and since they are used to implement the `transfer` entrypoint as well, I think there probably duplicate work between me and @PruStephan. 

So if Stepan has not work on that entrypoint yet, either I could work on it or me and Stepan could share code so that it would be easier to merge later on.

What do you think, Ivan?

Reply:
Well, my idea was to reuse `debitFrom` and `creditTo` from `morley-ledgers`, but for that we need https://gitlab.com/morley-framework/morley-ledgers/-/issues/10 and that's what Stepan is currently working on.

If everything else required for #14 is already implemented or blocked, you can implement `debitFrom` and `creditTo` in Lorentz (because eventually we'll need them implemented in Lorentz anyway). But if there is something else to do in #14, I think it's better to do that first.

Reply:
I see, that makes sense, Ivan. I am working on other aspect of `propose` and other entrypoints for now.